.github/workflows/compile-examples.yml

```diff
@@ -42,9 +42,23 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+      # for size delta comparison
+      - name: Fetch PR base branch
+        if: github.event_name == 'pull_request'
+        run: git fetch --no-tags --prune --depth=1 origin +${{ github.base_ref }}
+
```

Comment:
This can all be removed once https://github.com/arduino/actions/pull/31 is merged.

Reply:
The PR has been merged and I have removed the now unnecessary step from the workflow.